### ISSUE TITLE
Method SetJSONMarshaler is missing

### ISSUE THREAD

**Issue by vovean** [go-resty/resty#667]
On the latest resty version (v2.7.0) trying to invoke `SetJSONMarshaler` gives the following error: `type *resty.Client has no field or method SetJSONMarshaler`

**Comment by vovean** [comment]
I installed it by adding `require github.com/go-resty/resty/v2 v2.7.0` to my go.mod

**Comment by jeevatkm** [comment]
@vovean It is not available in v2.7.0. Those setter methods will be released with v2.8.0.

Currently, you could do -

For example:
```go
// Example of registering json-iterator
import jsoniter "github.com/json-iterator/go"

json := jsoniter.ConfigCompatibleWithStandardLibrary
client := resty.New()

client.JSONMarshal = json.Marshal
client.JSONUnmarshal = json.Unmarshal

```
